.github/workflows/format-code.yml

```
@@ -48,14 +43,7 @@ jobs:
           git config --global user.email "github-actions[bot]@users.noreply.github.com"
           git config --global user.name "github-actions[bot]"
 
-          # Commit the changes made by prettier
+          # Commit the changes made by dotnet format
```

Did you change this on the wrong branch?

```suggestion
          # Commit the changes made by prettier
```